### PR TITLE
bring us into 2015

### PR DESCRIPTION
**Pull request by jlipps**

### CODE REVIEW

**Comment by hackygolucky** [comment]
Well that's fun. It's amazing how many people can read something and miss this.  !m @jlipps

**Comment by jden** [comment]
We are living in the future!

**Comment by jlipps** [comment]
Oh man I hope not! teh_future

**Comment by jden** [comment]
back-to-the-future-gif-3458-19638-hd-wallpapers 1

**Comment by jlipps** [comment]
:+1:

**Comment by zachattack** [comment]
yess!

File: proposals/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Each day is organized around a theme. Share your expertise with the Cascadia com
...
+We invite the CSS and JavaScript community to submit talks for CJSFest 2015 at the Semiamhoo Resort in Washington. This Call for Proposals will close on March 15th. Diversity of voices is important to our community, so please don't think that you need to be a programming celebrity in order to speak at CJSFest 2015. The deadline to submit a proposal is **March 15, 2015 11:59pm UTC-1200**. If it's still March 15th, no matter where you are in the world, you still have time! When you’re ready to submit a proposal and you've read the instructions below that include the required template, go [HERE](https://github.com/cascadiajs/2015.cascadiajs.com/issues).
```
Comment (ryan-roemer) [inline proposals/README.md]: Just digging in again to my email backlog, so maybe I'm missing it -- but isn't `Semiahmoo` misspelled here? (search on `Semiamhoo`).

I know there are other PRs fixing this, so maybe I've missed it, but I think this is still in `master`.

**Pull request merged**